launch/dlio.launch

```diff
@@ -21,8 +21,12 @@
   <arg name="pointcloud_topic" default="/os_cloud_node/points"/>
   <arg name="imu_topic" default="/os_cloud_node/imu"/>
 
+  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_base_link_to_os_sensor"
+    args="0.5 0 1   1 0 0 0 base_link  os_sensor " />
```

Comment:
Where did you get the translations? Seem a bit arbitrary, did you take them from my slack message with all the coords from the cad?

Reply:
Yes, a rough approximation, it still worked. However this transform shouldn't be here but on slam_central, gonna update code